addons/auth_signup/data/mail_template_data.xml

```diff
@@ -341,5 +341,65 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Email template for new device login alert -->
+        <record id="alert_new_device_email" model="mail.template">
+            <field name="name">Settings: Alert New Device</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="subject">New Connection to your Account</field>
+            <field name="email_from">"{{ object.company_id.name }}" &lt;{{ (object.company_id.email or user.email) }}&gt;</field>
+            <field name="email_to">{{ object.email_formatted }}</field>
+            <field name="description">Sent to user after his account got logged into from an unknown device</field>
+            <field name="body_html" type="html">
+                <table border="0" cellpadding="0" cellspacing="0" style="padding-top: 16px; background-color: #FFFFFF; font-family:Verdana, Arial,sans-serif; color: #454748; width: 100%; border-collapse:separate;"><tr><td align="center">
```

Comment:
As you use anyway an email layout, is it required to have such content ? Can't you simplify the content ? Seems there is a lot of extra layouting, better keep content as simple as possible.